src/text/raw_text_writer.rs

```diff
@@ -737,6 +727,14 @@ impl<W: Write> IonWriter for RawTextWriter<W> {
         self.output.flush()?;
         Ok(())
     }
+
+    fn output(&self) -> &W {
+        self.output.get_ref()
+    }
+
+    fn output_mut(&mut self) -> &mut W {
+        self.output.get_mut()
+    }
```

Comment:
This looks like a move- I see the doc comments in the deleted code, are they preserved elsewhere? 

**EDIT**: Nevermind I see much more detailed commentary added in `writer.rs`.